evap/evaluation/templates/index.html

```diff
@@ -11,39 +9,45 @@
+                            {% csrf_token %}
+                            <input type="hidden" name="submit_type" value="login_username" />
+                            {% for field in login_username_form %}
+                                {% include "bootstrap_form_field.html" with field=field wide=True %}
+                            {% endfor %}
+                            <div class="form-submit">
+                                <div class="ml-auto col-sm-8">
+                                    <button type="submit" class="btn btn-primary">{% trans "Login" %}</button>
+                                </div>
+                            </div>
+                        </form>
                     </div>
-                </form>
+                </div>
             </div>
         </div>
-
         <div class="col-md-6">
-            <h2>{% trans "External and D-School users" %}</h2>
-            <div class="well">
-                <p>{% trans "Here you can request a one-time login URL. We will send it to your email address." %}</p>
-                <form class="form-horizontal" action="{% url "evaluation:index" %}{% if 'next' in request.GET %}?next={{ request.GET.next }}{% endif %}" method="POST">
-                    {% csrf_token %}
-                    <input type="hidden" name="submit_type" value="new_key" />
-                    {% bootstrap_field new_key_form.email layout='horizontal' horizontal_label_class='col-sm-3' horizontal_field_class='col-sm-9' placeholder=''%}
-                    <div class="form-group">
-                        <div class="col-sm-offset-3 col-sm-9">
-                            <input class="btn btn-default" type="submit" value="{% trans "Request login URL" %}">
-                        </div>
+            <div class="card">
+                <div class="card-body">
+                    <h2 class="card-title">{% trans "External and D-School users" %}</h2>
+                    <p class="card-subtitle mb-2 text-muted">{% trans "Here you can request a one-time login URL. We will send it to your email address." %}</p>
+                    <div class="card-text">
+                        <form class="form-horizontal" action="{% url "evaluation:index" %}{% if 'next' in request.GET %}?next={{ request.GET.next }}{% endif %}" method="POST">
+                            {% csrf_token %}
+                            <input type="hidden" name="submit_type" value="new_key" />
+                            {% include "bootstrap_form_field.html" with field=new_key_form.email wide=True %}
+                            <div class="form-submit">
+                                <div class="ml-auto col-sm-8">
+                                    <button type="submit" class="btn btn-secondary">{% trans "Request login URL" %}</button>
+                                </div>
+                            </div>
+                        </form>
                     </div>
-                </form>
+                </div>
```

Comment:
several cols on this page (maybe you just do a ctrl-f for that in the complete project)

Reply:
okay. we're keeping the cols for now where they currently are, as they are working in cases where the 12-part grid is sufficient. we're also keeping percentage values where i set them to have a more precise distribution.
when working on the mobile friendly stuff everything with columns has to be changed again (most parts somehow have to use the cols then) - regardless of what we're using now and currently i don't want to use the cols everywhere because it makes some larger tables even harder to layout than already the case.
so for now everything stays as it is.